code/model/jobs/DNDeployment.php

```diff
@@ -43,7 +43,8 @@ class DNDeployment extends DataObject implements Finite\StatefulInterface, HasSt
 		"Branch" => "Varchar(255)",
 		"State" => "Enum('New, Submitted, Invalid, Queued, Deploying, Aborting, Completed, Failed', 'New')",
 		// JSON serialised DeploymentStrategy.
-		"Strategy" => "Text"
+		"Strategy" => "Text",
+		"DeploymentDate" => "SS_Datetime"
```

Comment:
I'd name it something like "DatePlanned", to distinguish from the actual deployment date?